doc/pipeline_spec.md

```diff
@@ -81,7 +156,7 @@ If a method is not specified, the "map" method is used by default.
 }
 ```
 
-This pipeline runs when the repo `my-input` gets a new commit.  The pipeline will spawn 4 parallel jobs, each of which runs the command `my-binary` in the Docker image `my-image`, with `arg1` and `arg2` as arguments to the command and `my-std-input` as the standard input.  Each job will get a part of the new commit as input because `method` is set to `map`.
+This pipeline runs when the repo `my-input` gets a new commit.  The pipeline will spawn 4 parallel jobs, each of which runs the command `my-binary` in the Docker image `my-image`, with `arg1` and `arg2` as arguments to the command and `my-std-input` as the standard input.  Each job will get a set of blocks from the new commit as its input because `method` is set to `map`.
 
 ## Accessing the output of a job's parent
```

Comment:
We need more details on pfs/prev (if we have it at all).  Additional details about when /prev is available (only when incrementality=true?) and more indepth about the "sum" example would help.